src/requirements/Twitter/components/TwitterVerificationSelect.tsx

```diff
@@ -0,0 +1,54 @@
+import { FormControl, FormLabel } from "@chakra-ui/react"
+import ControlledSelect from "components/common/ControlledSelect"
+import { useFormContext } from "react-hook-form"
+import { RequirementFormProps } from "requirements"
+import { SelectOption } from "types"
+import parseFromObject from "utils/parseFromObject"
+
+type ValueType = "any" | "blue" | "business" | "government"
+
+const options: SelectOption<ValueType>[] = [
+  {
+    label: "One of them",
+    value: "any",
+  },
+  {
+    label: "Blue",
+    value: "blue",
+  },
+  {
+    label: "Business",
+    value: "business",
+  },
+  {
+    label: "Government",
+    value: "government",
+  },
+]
+
+const TwitterVerificationSelect = ({
+  baseFieldPath,
+}: RequirementFormProps): JSX.Element => {
+  const {
+    setValue,
+    formState: { errors },
+  } = useFormContext()
+  const defaultValue: ValueType = "any"
+
+  return (
+    <FormControl isInvalid={!!parseFromObject(errors, baseFieldPath)?.type}>
+      <FormLabel>Verification type</FormLabel>
+
+      <ControlledSelect
```

Comment:
You can use this component as you'd use a react-hook-form `Controller`, so you just need to pass a name, defaultValue, options, and no need to handle onChange ;) 

I'm pretty sure you won't event need to handle that "any" value, you can just set the first option's value to `null` and it'll work out of the box!

Reply:
<img width="498" alt="image" src="https://github.com/guildxyz/guild.xyz/assets/66187782/8184cdf5-0eab-454a-ab97-52411ad51d1f">
I've got this message when I changed it to null, that's because I chosen a more readable value.

Reply:
I change it to null, and add a todo somewhere